trin-core/src/portalnet/discovery.rs

```diff
@@ -105,19 +107,13 @@ impl Discovery {
             builder.build(&enr_key).unwrap()
         };
 
-        info!(
-            "Starting discv5 with local enr encoded={:?} decoded={}",
-            enr, enr
-        );
-
         let discv5 = Discv5::new(enr, enr_key, config.discv5_config)
             .map_err(|e| format!("Failed to create discv5 instance: {}", e))?;
 
         for enr in config.bootnode_enrs {
-            info!("Adding bootnode {}", enr);
```

Comment:
There's probably some value in logging the bootnode IP addresses at an info level, which seems to get lost in this change.

Reply:
I think it's okay to keep this removed given that both the [history](https://github.com/ethereum/trin/blob/765d75b0ecc7986e678b2f260477aac1afd39369/trin-history/src/lib.rs#L85) and [state](https://github.com/ethereum/trin/blob/765d75b0ecc7986e678b2f260477aac1afd39369/trin-state/src/lib.rs#L79) network initializations log the boot nodes.

Reply:
Yup, my issue was specifically that those logs don't include the IP addresses.

Reply:
Oh whoops I totally overlooked that. Created #430.